apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,245 @@
+                            opacity="0"
+                            w="100%"
+                            h="100%"
+                            position="absolute"
+                            top="0"
+                            left="0"
+                            justify="center"
+                            gap="0.25rem"
+                            color="base.content.brand"
+                            _groupHover={{
+                              opacity: 1,
+                            }}
+                            as="button"
+                          >
+                            <Icon as={BiShow} />
+                            <Text textStyle="caption-1">Click to preview</Text>
+                          </VStack>
+                        </>
+                      )}
+                    </Box>
+                    <HStack w="100%" justifyContent="space-between">
+                      <Text
+                        textAlign="left"
+                        textStyle="h6"
+                        textColor="base.content.strong"
+                        data-active={dataAttr(
+                          selectedLayout.layoutTypename ===
+                            layoutEntry.layoutTypename,
+                        )}
+                        _active={{ textColor: 'base.content.brand' }}
+                        mb="0.5rem"
+                        transition="color 300ms ease-out"
+                        _groupHover={{ textColor: 'base.content.brand' }}
+                      >
+                        {layoutEntry.layoutDisplayName} layout
+                      </Text>
+                    </HStack>
+                    <Text
+                      w="100%"
+                      textAlign="left"
+                      textStyle="body-1"
+                      textColor="base.content.default"
+                    >
+                      {layoutEntry.layoutDescription}
+                    </Text>
+                  </VStack>
+                )
+              })}
+            </VStack>
+          </GridItem>
+          <GridItem
+            w="100%"
+            colSpan={3}
+            bg="grey.100"
+            p="2rem 2rem 0 2rem"
+            overflow="auto"
+          >
+            <Box
+              borderTopRadius="8px"
+              w="100%"
+              minH="100%"
+              boxShadow="md"
+              position="relative"
+            >
+              <HStack
+                borderTopRadius="8px"
+                w="100%"
+                bgColor="base.canvas.light"
+                textColor="white"
+                justifyContent="center"
+                gap="0"
+                padding="0.5rem 0.75rem"
+              >
+                <Text textStyle="caption-2">
+                  {`You're previewing the`}&nbsp;
+                </Text>
+                <Text textStyle="caption-1">
+                  {selectedLayout.layoutDisplayName}
+                  {` Layout`}
```

Comment:
nit: unnecessary templating, or if you want you can consolidate them

```suggestion
                  {`${selectedLayout.layoutDisplayName} Layout`}
```